libs/blocks/hero-marquee/hero-marquee.js

```diff
@@ -132,6 +133,21 @@ function parseKeyString(str) {
   return result;
 }
 
+function isElementEmpty(element) {
```

Comment:
It's very likely I'm missing something, but couldn't we simply check for `element.textContent.trim()`? I'd assume that if there's some text anywhere, the element is not empty.

Reply:
@overmyheadandbody  - Most of the time this row has a <picture> or <video> element and no textContent. This also uses per-viewport background cols that have similar conditions. That's why i've done the overly verbose conditions here, but lmk if you see a slimmer way to handle that.

Reply:
Since we know of these `<picture>` and `<video>` use-cases, couldn't we do something like
```suggestion
  return element.textContent.trim() && !element.querySelector('source, src');
```

Sorry if I'm a bit dense about this, this just seems more complicated than needed to me. If you have other reasons why this wouldn't work we can move forward with this.

Reply:
Just an FYI depending on the timing of this, the 'source' is not available // but added via an intersection observer.  So it might be safer to check if it has a `video` element

Reply:
Thanks for the suggestion. I was overthinking it and yea since we know these are either text, video, picture etc this makes sense to just query that. I've updated the logic on that and just removed that isElementEmpty() func. 
I tested all the pages and it checks out.

Reply:
Great, thanks! Glad this could be simplified!